apps/src/templates/sectionProgress/SectionProgress.jsx

```diff
@@ -81,6 +89,9 @@ class SectionProgress extends Component {
 
     const levelDataInitialized = scriptData && studentLevelProgress;
 
+    const lessonNumbers = scriptData ?
+      Array(scriptData.stages.length).fill().map((e,i)=>i+1) : [];
```

Comment:
since we use lodash, I think it'd be cleaner to use `range` here: https://lodash.com/docs#range

you can import lodash like this:
https://github.com/code-dot-org/code-dot-org/blob/77eade273eed29c48691f4e6b894b69d5c83cc15/apps/test/unit/code-studio/pd/professional_learning_landing/plcElements/moduleAssignmentTest.js#L2